### PR TITLE
refactor: some prep for starting the reviewer port

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Approach

In the commits

---

Trusting CI as tests

### CODE REVIEW

File: AnkiDroid/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -428,7 +428,7 @@
             android:configChanges="orientation|screenSize"
             />
         <activity
-            android:name="com.ichi2.anki.previewer.PreviewerActivity"
+            android:name="com.ichi2.anki.previewer.CardViewerActivity"
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/AndroidManifest.xml:431]: I should change the package as well, but I don't want to think about that now

Comment (david-allison) [reply AnkiDroid/src/main/AndroidManifest.xml:431]: If it's `exported=false`, it's not our public API, so we have time

File: AnkiDroid/src/main/java/com/ichi2/anki/Ease.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import androidx.annotation.IntDef
+
+@IntDef(AGAIN, HARD, GOOD, EASY)
+annotation class Ease
+
+const val AGAIN = 1
+const val HARD = 2
+const val GOOD = 3
+const val EASY = 4
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/Ease.kt:26]: Would you consider making this an enum?

`IntDef` is from that ridiculous point in Android history where enums were considered to be too 'heavy' as they were backed by classes

**Review by david-allison — APPROVED** [review]

**Review by lukstbit — APPROVED** [review]
LGTM, with a plus for using an enum.

**Pull request merged**